apps/coscrad-frontend/src/store/slices/ResourcesSlice.ts

```diff
@@ -0,0 +1,69 @@
+import { IAggregateInfo } from '@coscrad/api-interfaces';
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import { getConfig } from '../../config';
+
+const RESOURCES = 'resources';
+
+export type ResourcesState = {
+    infos: IAggregateInfo[];
+    resources: unknown[];
+    isLoading: boolean;
+};
+
+const initialState: ResourcesState = {
+    // We may want a separate resourceInfoSlice
+    infos: [],
+    resources: [],
+    isLoading: false,
+};
+
+// TODO move to api-interfaces
+type FSA<TPayload = unknown> = {
+    type: string;
+    payload: TPayload;
+};
+
+// we might want to type this and keep it as a helper for testing \ sandboxing
+const delay = async (t, v) => new Promise((resolve) => setTimeout(resolve.bind(null, v), t));
+
+export const fetchResourceInfos = createAsyncThunk(`${RESOURCES}/fetchInfos`, async () => {
+    console.log('initialting requestt!');
```

Comment:
(sp) requestt - I presume this isn't a long term line any way....

Reply:
I need to remove the log.

Reply:
done